### PR TITLE
'load more' button CSS improvement

### PR DESCRIPTION
**Pull request by mmmavis**
@alanmoo I thought about doing the suggestion you have on https://github.com/mozilla/network-pulse/pull/348#discussion_r100635697 but that means we are wrapping all project cards in ONE `.row` element... which doesn't sound right to me? If we wanna make use of `.row` class we should wrap projects in groups of 3 (can be tackled as a separate PR if needed).

### CODE REVIEW

**Comment by mmmavis** [comment]
PR updated!

File: scss/main.scss [hunk 1]
```diff
@@ -31,38 +31,17 @@ p {
 }
 
 .btn-link {
-  cursor: pointer;
-  padding: 0;
+  &.inline-link {
```
Comment (alanmoo) [inline scss/main.scss]: Doesn't this just make it look like a regular anchor element? Can we just use `<a href="/">` with no classes?

Comment (mmmavis) [inline scss/main.scss]: We should only use `<a>` for hyperlinks. I can rename the class name if it's confusing.

Comment (alanmoo) [inline scss/main.scss]: While I think this could be a hyperlink, this is fine, let's get it merged in. We've got bigger things to deal with.

**Pull request merged**